.detoxrc.js

```diff
@@ -84,7 +84,7 @@ module.exports = {
     },
     'android.emu.x86_64': {
       device: 'emulator_x86_64',
-      app: 'android.debug'
+      app: 'Pixel_2_API_30'
```

Comment:
We need this? the user have to have this specific device in the local env.... mmmm, I don't think so.